queue_test.go

```diff
@@ -2,9 +2,10 @@ package events
 
 import (
 	"fmt"
-	"sync"
 	"testing"
 	"time"
+
+	"golang.org/x/sync/errgroup"
```

Comment:
Oh, right; if we must have to  use this one, then I guess we'd not be able to have zero-dependencies.

Reply:
Maybe I can take a second try at this. Was only working to appease the linter. Might not be worth pulling in a dependency just for that.